doc/classes/AudioStreamPlayer.xml

```diff
@@ -111,26 +115,33 @@
 			<return type="void">
 			</return>
 			<description>
+				Stops the audio.
 			</description>
 		</method>
 	</methods>
 	<members>
 		<member name="autoplay" type="bool" setter="set_autoplay" getter="is_autoplay_enabled">
+			If [code]'true'[/code], audio plays from start of the game.
```

Comment:
Not at the start of the game - it plays when the node enters the scene tree.  Also add default value.

Reply:
Please remove single quotes inside [code] tag too.